inc/saiport.h

```diff
@@ -1214,6 +1261,33 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_PTP_MODE,
 
+    /**
+     * @brief Port serdes control attributes start
+     *
+     * Start of port serdes control values. The values are of type sai_u32_list_t
+     * where the count is number lanes in a port and the list specifies list of values
+     * to be applied to each lane.
+     *
+     * @type sai_u32_list_t
+     * @flags CREATE_AND_SET
+     * @default internal
+     * @range SAI_PORT_ATTR_SERDES_RANGE
+     */
+    SAI_PORT_ATTR_SERDES_MIN,
```

Comment:
1. It might be good to have a "struct" type defined for the "list of different pre-emphasis taps" and add it into the sai_attribute_value_t union. 
2. Rename the attribute to denote TX_SERDES_PARAMS or TX_PRE_EMPHASIS